tests/test_plugins.py

```diff
@@ -557,6 +568,12 @@ def take_action(self, alert, action, text, **kwargs):
 
         return alert, action, text
 
+    def post_take_action(self, alert, action, text, **kwargs):
+
+        if action == 'test-post-action':
+            alert.tags.append('test-post-action')
```

Comment:
It would be better to have something a bit more realistic as the `post_action` hook. What is a use-case that you currently have for post actions? Does it fit with this example of "createTicket", "updateTicket", "resolveTicket"?

Reply:
My use case is actually that it triggers some external event. In our case its pushing a new event to a kafka topic that the "action" was completed, so I did not really want to put that into the test-case. I just wanted to make sure that the function was triggered.